### PR TITLE
Add DISK field to biosnoop

### PR DESCRIPTION
**Pull request by DavidVentura**
Add disk field to biosnoop.

I tried to also add `req->part->part_no` but I felt I was not getting what I expected on it when writing to certain partitions. 

##### Checklist

- [ ] Language changes are updated in `docs/reference_guide.md`
- [X] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [ ] The new behaviour is covered by tests

### CODE REVIEW

**Comment by brendangregg** [comment]
Thanks for testing on different platforms. I think the field should be 7-chars wide to match the BCC version, plus as this grows to approach the BCC version (if it does) we should keep the default output <80 chars.

File: tools/biosnoop.bt [hunk 1]
```diff
@@ -12,32 +13,36 @@
...
 /@start[arg0] != 0 && @iopid[arg0] != 0 && @iocomm[arg0] != ""/
+
 {
 	$now = nsecs;
-	printf("%-12u %-16s %-6d %7d\n",
-	    elapsed / 1e6, @iocomm[arg0], @iopid[arg0],
-	    ($now - @start[arg0]) / 1e6);
+	printf("%-12u %-7s %-16s %-6d %7d\n",
+	    elapsed / 1000000, @disk[arg0], @iocomm[arg0], @iopid[arg0],
+	    ($now - @start[arg0]) / 1000000);
```
Comment (danobi) [inline tools/biosnoop.bt:34]: What's the reason for the `1e6` -> `1000000` changes?

Comment (DavidVentura) [reply tools/biosnoop.bt:34]: my compiler refused to parse 1e6 -- i'll revert this change if you think i should

Comment (danobi) [reply tools/biosnoop.bt:34]: You're probably on an older bpftrace. Scientific notation was merged in #1476.

Upstream tools should be targeting upstream bpftrace - we don't guarantee upstream tools are compatible across old versions of bpftrace.

**Review by danobi — APPROVED** [review]
Thanks!

**Pull request merged**